### PR TITLE
C code generation visitor

### PR DESCRIPTION
**Pull request by Cklein17**

### CODE REVIEW

File: src/main/code.ezuino [hunk 1]
```diff
@@ -1,6 +1,5 @@
-func int main() {
-    while(1<2){
-        int a
-    }
-    return 1
+int a
```
Comment (sentoa) [inline src/main/code.ezuino]: Don't include code.ezuino

File: src/main/java/ast/Func_defNode.java [hunk 2]
```diff
@@ -1,64 +1,65 @@
...
-	
-	@Override
-	public void acceptLevel(AstLevelVisitor v, int level) {
-		v.visitLevel(this, level);
-	}
+    @Override
+    public String toString() {
+        return "Func_defNode{" +
+                "ID='" + ID + '\'' +
+                '}';
```
Comment (joandrsn) [inline src/main/java/ast/Func_defNode.java]: Missing `type` - Wrong merge?

File: src/main/java/ezuino/Main.java [hunk 3]
```diff
@@ -97,6 +98,10 @@ public static void main(String[] args) throws IOException {
         ListVisitor lv = new ListVisitor(errorhandler);
         astNode.accept(lv);
 
+        CCodeGenerationVisitor cCodeGenerationVisitor = new CCodeGenerationVisitor(System.out);
+        astNode.accept(cCodeGenerationVisitor);
+
+        // ErrorHandler.printErrorList();
```
Comment (joandrsn) [inline src/main/java/ezuino/Main.java]: Remove old code

File: src/main/code.ezuino [hunk 4]
```diff
@@ -1,6 +1,5 @@
...
+func  b(int a) {
+
 }
+a := b(1)
```
Comment (joandrsn) [inline src/main/code.ezuino]: Remove this file from commits
Use 
* `git fetch origin` 
* `git checkout origin/master -- src/main/code.ezuino` 
And commit the file

File: src/main/java/ezuino/Ezuino.g4 [hunk 5]
```diff
@@ -108,7 +108,7 @@ INTDCL              : 'int' ;
 DOUBLEDCL           : 'double' ;
 STRINGDCL           : 'string' ;
 BOOLEANDCL          : 'boolean' ;
-LISTDCL                : 'list' ;
+LISTDCL             : 'list' ;
```
Comment (sentoa) [inline src/main/java/ezuino/Ezuino.g4]: Remove list from gramma

File: src/main/java/ezuino/Main.java (hunk 3 above)
Comment (sentoa) [inline src/main/java/ezuino/Main.java]: Remove comment

**Pull request merged**